stylesheet.css

```diff
@@ -36,4 +36,8 @@
 
 .action-stats {
 	color: greenyellow;
-}
\ No newline at end of file
+}
+
+.button:checked {
+  color: red;
```

Comment:
That's a placeholder, no?

Reply:
Jakub opened https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/4757 now.

Reply:
indeed a placeholder. I'm mentioning that issue in a comment.